wire-ios-request-strategy/Sources/Request Strategies/Feature configurations/GetFeatureConfigsActionHandler.swift

```diff
@@ -66,11 +66,18 @@ final class GetFeatureConfigsActionHandler: ActionHandler<GetFeatureConfigsActio
                         data: data,
                         repository: repository
                     )
-                case .v6, .v7, .v8:
+
+                case .v6, .v7:
                     try processor.processActionPayloadAPIV6(
                         data: data,
                         repository: repository
                     )
+
+                case .v8:
+                    try processor.processActionPayloadAPIV8(
```

Comment:
I wasn't sure it requires a new dedicated process method here, @KaterinaWire what do you think?

Reply:
No, you don't need to add a new one. Anyway "if let channels = payload.channels {...}" is optional.
We have `processActionPayload` and `processActionPayloadAPIV6` because the structure of the `conferenceCalling` feature was changed in v6.

Reply:
ok i'll make the change then